examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,566 @@
+    "pyplot.xlabel('X')\n",
+    "pyplot.ylabel('Y');"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from devito import Grid\n",
+    "\n",
+    "grid = Grid(shape=(nx, ny), extent=(2., 2.))\n",
+    "x, y = grid.dimensions\n",
+    "t = grid.stepping_dim\n",
+    "\n",
+    "u = numpy.zeros((nx, ny))\n",
+    "v = numpy.zeros((nx, ny))\n",
+    "p = numpy.zeros((nx, ny)) \n",
+    "b = numpy.zeros((nx, ny))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Reminder: here are our equations\n",
+    "\n",
+    "$$ \\frac{\\partial u}{\\partial t} + u \\frac{\\partial u}{\\partial x} + v \\frac{\\partial u}{\\partial y}= -\\frac{1}{\\rho}\\frac{\\partial p}{\\partial x}  + \\nu \\left[ \\frac{\\partial^2 u}{\\partial x^2} +\\frac{\\partial^2 u}{\\partial y^2} \\right] $$\n",
+    "\n",
+    "$$ \\frac{\\partial v}{\\partial t} + u \\frac{\\partial v}{\\partial x} + v \\frac{\\partial v}{\\partial y}= -\\frac{1}{\\rho}\\frac{\\partial p}{\\partial y}  + \\nu \\left[ \\frac{\\partial^2 v}{\\partial x^2} +\\frac{\\partial^2 v}{\\partial y^2} \\right] $$\n",
+    "\n",
+    "$$\n",
+    "\\frac{\\partial^2 p}{\\partial x^2} +\\frac{\\partial^2 p}{\\partial y^2} =\n",
+    "\\rho \\left[\\frac{\\partial}{\\partial t} \\left(\\frac{\\partial u}{\\partial x} + \\frac{\\partial v}{\\partial y} \\right) - \\left(\\frac{\\partial u}{\\partial x}\\frac{\\partial u}{\\partial x}+2\\frac{\\partial u}{\\partial y}\\frac{\\partial v}{\\partial x}+\\frac{\\partial v}{\\partial y}\\frac{\\partial v}{\\partial y} \\right) \\right]\n",
+    "$$\n",
+    "\n",
+    "Note that p has no time dependence, so we are going to solve for p  in pseudotime then step to then next time step and solve for u and v. This will require two operators, one for p (using p and pn) in  pseudotime and one for u and v in time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from devito import TimeFunction, first_derivative, Function, \\\n",
+    "Constant, Eq, solve, left, Operator, configuration\n",
+    "\n",
+    "# Build Required Functions and derivatives:\n",
+    "# --------------------------------------\n",
+    "# |Variable | Required Derivatives     |\n",
+    "# --------------------------------------\n",
+    "# |   u     | dt, dx, dy, dx**2, dy**2 |\n",
+    "# |   v     | dt, dx, dy, dx**2, dy**2 |\n",
+    "# |   p     |     dx, dy, dx**2, dy**2 |\n",
+    "# |   pn    |     dx, dy, dx**2, dy**2 |\n",
+    "# --------------------------------------\n",
+    "u = TimeFunction(name='u', grid=grid, space_order=2)\n",
+    "u_dx = first_derivative(u, dim=x)\n",
+    "u_dy = first_derivative(u, dim=y)\n",
+    "\n",
+    "v = TimeFunction(name='v', grid=grid, space_order=2)\n",
+    "v_dx = first_derivative(v, dim=x)\n",
+    "v_dy = first_derivative(v, dim=y)\n",
```

Comment:
Should I have a comment here highlighting this? This was explicitly requested the other day in slack.

Reply:
Yes please add a comment, I personally think this should be removed since the shortcuts (`.dx, .dy, ...`) are used for the equations.